frontend/src/data/theads.ts

```diff
@@ -0,0 +1,37 @@
+import { IThead } from './interfaces';
```

Comment:
as mentioned in a previous comment, no need for separate data file for table head. please remove this file